python/test/t_RandomMixture_std.py

```diff
@@ -329,15 +309,20 @@
     maxS = 10.0
     muS = (log(minS) + log(maxS)) / 2.0
     sigma = (log(maxS) - muS) / 3.0
-    atom1 = TruncatedDistribution(LogNormal(muS, sigma), minS, maxS)
-    atom2 = Uniform(0.0, 2.0)
+    atom1 = ot.TruncatedDistribution(ot.LogNormal(muS, sigma), minS, maxS)
+    atom2 = ot.Uniform(0.0, 2.0)
     sum = atom1 + atom2
     print("sum=", sum)
     print("CDF=%.6g" % sum.computeCDF(2.0))
     print("quantile=", sum.computeQuantile(0.2))
     # For ticket 1129
-    dist = RandomMixture([Uniform()]*200)
+    dist = ot.RandomMixture([ot.Uniform()]*200)
     print("CDF(0)=%.5g" % dist.computeCDF([0]))
-except:
-    import sys
-    print("t_RandomMixture_std.py", sys.exc_info()[0], sys.exc_info()[1])
+
+# check parameter accessors
+dist = ot.Gumbel() + ot.Normal(0, 0.1)
```

Comment:
Are there examples of the construction with the overloaded `+` in the doc? It's great!

Reply:
there is no dedicated example, but 2 mention it, look for "arithmetic" in python/doc/examples

Reply:
Thanks! Maybe we could also add an example to the [RandomMixture](https://openturns.github.io/openturns/1.16/user_manual/_generated/openturns.RandomMixture.html) API page.